jobs/UnitTest/PRGateUnitTest.groovy

```diff
@@ -0,0 +1,51 @@
+String stash_manifest_name
```

Comment:
Can you add comments either in PR or code to describe the class/groovy releationship?